src/main/java/com/hierynomus/smbj/auth/NtlmAuthenticator.java

```diff
@@ -88,6 +88,8 @@ public AuthenticateResponse authenticate(final AuthenticationContext context, fi
                 logger.debug("Received NTLM challenge from: {}", challenge.getTargetName());
 
                 response.setWindowsVersion(challenge.getVersion());
+                response.setNetBIOSName(String.valueOf(challenge.getAvPairObject(AvId.MsvAvNbComputerName)));
```

Comment:
This will convert `null` to `"null"`. I'd suggest adding a helper such as `getAvPairString` which will return null instead.